### DISCUSSION TITLE
Help switching between floating and tiling modes

### DISCUSSION THREAD

**Discussion by chargedcs** [qtile/qtile#3522] [Q&A] [answered]
# Help switching between floating and tiling modes
Hey there, I recently started to use Qtile, and I learnt that holding modkey and dragging with your mouse enables floating mode for the window, I have looked everywhere I could think of, but I couldn't find a way to **bring that same window back into tiling mode**. Does anyone know how to do so? Any help will be appreciated.

**Comment by m-col** [comment, marked as answer]
You'll need to define a key binding for `lazy.window.toggle_floating()`, calling that will toggle the float state of the current window

**Reply by chargedcs** [reply]
Tysm, that worked :)
